task/net/src/bsp/sidecar_1.rs

```diff
@@ -22,7 +22,7 @@ enum Trace {
     KszId(u16),
     Vsc8552Status { port: u8, status: u16 },
 }
-ringbuf!(Trace, 16, Trace::None);
+ringbuf!(SIDECAR_RINGBUF, Trace, 16, Trace::None);
```

Comment:
I'm confused about this part of the change, can you describe what happened without this? It may point to a bug in the ringbuf macros.

Reply:
I understood that sidecar.rs and main.rs can't both have unnamed rings, but maybe that's wrong.